lib/facebook_app_events.dart

```diff
@@ -176,7 +197,7 @@ class FacebookAppEvents {
   ///
   /// See: https://developers.facebook.com/docs/reference/androidsdk/current/facebook/com/facebook/appevents/appeventsconstants.html/#eventnameviewedcontent
   Future<void> logViewContent({
-    Map<String, dynamic> content,
+    String content,
     String id,
     String type,
```

Comment:
Hm, did this change in some new version of Facebook SDK? I see that I added it as a `Map<String, dynamic>` at some point but found this at the latest documentation.

```java
/**
 * This function assumes logger is an instance of AppEventsLogger and has been
 * created using AppEventsLogger.newLogger() call.
 */
public void logViewContentEvent (
String contentType, 
String contentData, 
String contentId, 
String currency, 
double price) {
    Bundle params = new Bundle();
    params.putString(AppEventsConstants.EVENT_PARAM_CONTENT_TYPE, contentType);
    params.putString(AppEventsConstants.EVENT_PARAM_CONTENT, contentData);
    params.putString(AppEventsConstants.EVENT_PARAM_CONTENT_ID, contentId);
    params.putString(AppEventsConstants.EVENT_PARAM_CURRENCY, currency);
    logger.logEvent(AppEventsConstants.EVENT_NAME_VIEWED_CONTENT, price, params);
}
```

It seems to be more than just the `Map` changing to `String` ... please apply the full change if making change to this.